internal/csi/csi.go

```diff
@@ -71,3 +82,73 @@ func BinsForRange(start, end uint32, minShift, depth int32) []uint16 {
 func maximumBinWidth(minShift, depth int32) uint32 {
 	return uint32(1 << uint32(minShift+depth*3))
 }
+
+// Read reads index data from csi and returns a set of BGZF chunks covering
```

Comment:
csiFile -> csi (it's a Reader, not a file)

Or just 'r' is fine too

Reply:
done